tools/docker/syzbot/Dockerfile

```diff
@@ -30,24 +30,15 @@ RUN curl https://dl.google.com/go/go1.22.7.linux-$(uname -m | sed 's/aarch64/arm
 ENV PATH /usr/local/go/bin:$PATH
 
 # The default clang-14 is too old, install the latest one.
```

Comment:
While at it, can you please bump the Clang version to the latest stable one?
(In a separate patch, so that we can isolate the problem if it arises)

Reply:
Doesn't matter :)
If the new version is broken, we just need to be able to check whether the breakage was introduce by the switch to archives, or by the version bump.

Reply:
After this patch, I will bump the version `ARG LLVM_VERSION=llvm-15.0.7-x86_64` to the latest `llvm-19.1.2-x86_64.tar.gz`, hopefully, it should work without issue.

Reply:
Checked `ARG LLVM_VERSION=llvm-19.1.2-x86_64`. Need to also update clang-format, clang-tidy apt packages in next patch, so should be good:

```
$ docker run --rm --name syz_master_diff -it syz_master_diff bash

/🤖 for e in {clang,clang++,clang-tidy,clang-format,ld.lld,llvm-nm,llvm-ar,llvm-objcopy,llvm-objdump,llvm-addr2line,llvm-readelf,llvm-strip}; do echo; $e --version; done

ClangBuiltLinux clang version 19.1.2 (https://github.com/llvm/llvm-project.git 7ba7d8e2f7b6445b60679da826210cdde29eaf8b)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /usr/bin

ClangBuiltLinux clang version 19.1.2 (https://github.com/llvm/llvm-project.git 7ba7d8e2f7b6445b60679da826210cdde29eaf8b)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /usr/bin

Debian LLVM version 15.0.6
  Optimized build.
  Default target: x86_64-pc-linux-gnu
  Host CPU: skylake

Debian clang-format version 15.0.6

ClangBuiltLinux LLD 19.1.2 (https://github.com/llvm/llvm-project.git 7ba7d8e2f7b6445b60679da826210cdde29eaf8b) (compatible with GNU linkers)

llvm-nm, compatible with GNU nm
LLVM (http://llvm.org/):
  LLVM version 19.1.2
  Optimized build.

LLVM (http://llvm.org/):
  LLVM version 19.1.2
  Optimized build.

llvm-objcopy, compatible with GNU objcopy
LLVM (http://llvm.org/):
  LLVM version 19.1.2
  Optimized build.

LLVM (http://llvm.org/):
  LLVM version 19.1.2
  Optimized build.


  Registered Targets:
    aarch64     - AArch64 (little endian)
    aarch64_32  - AArch64 (little endian ILP32)
    aarch64_be  - AArch64 (big endian)
    arm         - ARM
    arm64       - ARM64 (little endian)
    arm64_32    - ARM64 (little endian ILP32)
    armeb       - ARM (big endian)
    bpf         - BPF (host endian)
    bpfeb       - BPF (big endian)
    bpfel       - BPF (little endian)
    hexagon     - Hexagon
    loongarch32 - 32-bit LoongArch
    loongarch64 - 64-bit LoongArch
    mips        - MIPS (32-bit big endian)
    mips64      - MIPS (64-bit big endian)
    mips64el    - MIPS (64-bit little endian)
    mipsel      - MIPS (32-bit little endian)
    ppc32       - PowerPC 32
    ppc32le     - PowerPC 32 LE
    ppc64       - PowerPC 64
    ppc64le     - PowerPC 64 LE
    riscv32     - 32-bit RISC-V
    riscv64     - 64-bit RISC-V
    systemz     - SystemZ
    thumb       - Thumb
    thumbeb     - Thumb (big endian)
    x86         - 32-bit X86: Pentium-Pro and above
    x86-64      - 64-bit X86: EM64T and AMD64

llvm-addr2line
LLVM (http://llvm.org/):
  LLVM version 19.1.2
  Optimized build.

LLVM (http://llvm.org/):
  LLVM version 19.1.2
  Optimized build.

llvm-strip, compatible with GNU strip
LLVM (http://llvm.org/):
  LLVM version 19.1.2
  Optimized build.

/🤖

```